R/zchunk_L232.industry.R

```diff
@@ -89,295 +80,554 @@ module_energy_L232.industry_DISABLED <- function(command, ...) {
     L123.in_EJ_R_indchp_F_Yh <- get_data(all_data, "L123.in_EJ_R_indchp_F_Yh")
     L1322.in_EJ_R_indenergy_F_Yh <- get_data(all_data, "L1322.in_EJ_R_indenergy_F_Yh")
     L1322.in_EJ_R_indfeed_F_Yh <- get_data(all_data, "L1322.in_EJ_R_indfeed_F_Yh")
-    A32.inc_elas_output <- get_data(all_data, "socioeconomics/A32.inc_elas_output")
-    L101.Pop_thous_GCAM3_R_Y <- get_data(all_data, "L101.Pop_thous_GCAM3_R_Y")
-    L102.pcgdp_thous90USD_GCAM3_R_Y <- get_data(all_data, "L102.pcgdp_thous90USD_GCAM3_R_Y")
-    L102.pcgdp_thous90USD_Scen_R_Y <- get_data(all_data, "L102.pcgdp_thous90USD_Scen_R_Y")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+    # 0. Give binding for variable names used in pipeline
+    has_district_heat <- sector <- fuel <- supplysector <- subsector <-
+      technology <- year.fillout <- to.value <- year <- share.weight <-
+      efficiency <- minicam.energy.input <- secondary.output <- coefficient <-
+      elec_ratio <- output.ratio <- . <- year.x <- output.ratio.x <- output.ratio.y <-
+      input.cost <- minicam.non.energy.input <- GCAM_region_ID <- value <-
+      calibrated.value <- sector.name <- subsector.name <- region <-
+      calOutputValue <- subs.share.weight <- calOutputValue.x <- calOutputValue.y <-
+      output_tot <- value.x <- value.y <- total <- fuelprefElasticity <- NULL
+
     # ===================================================
+    # 1. Perform computations
+    # Create tables to delete technologies and subsectors in regions where heat is not modeled as a fuel
+    has_not_heat <- filter(A_regions, has_district_heat == 0) # intermediate tibble
```

Comment:
This is a much more succinct way to distinguish regions with and without district heat than the old code.  But with the old code.  Could you add more comments as to what is being done and why?